tensor2tensor/rl/envs/batch_env.py

```diff
@@ -43,11 +43,12 @@ def __init__(self, envs, blocking):
     """
     self._envs = envs
     self._blocking = blocking
-    observ_space = self._envs[0].observation_space
```

Comment:
If we're using only the dtype externally to this code block, please make only that part of the object to limit leakage. I.e.,

observ_space = self._envs[0].observation_space
self.observ_dtype = observ_space.dtype

Reply:
We're using the whole `observation_space` in InGraphBatchEnv, as it's passed do `utils.parse_{dtype,shape}` to determine observation datatype and shape.